src/libraries/System.Net.NetworkInformation/src/System.Net.NetworkInformation.csproj

```diff
@@ -248,11 +251,11 @@
     <Reference Include="System.Threading.Overlapped" />
     <Reference Include="System.Threading.ThreadPool" />
   </ItemGroup>
-  <ItemGroup Condition="'$(TargetsUnix)' == 'true' or '$(TargetsBrowser)' == 'true'">
+  <ItemGroup Condition="'$(TargetsUnix)' == 'true' and '$(TargetsBrowser)' != 'true'">
```

Comment:
The `TargetsBrowser` part of the and is not needed anymore.

Reply:
Fixed